store/postgres/src/relational_queries.rs

```diff
@@ -3511,6 +3556,24 @@ impl<'a> QueryFragment<Pg> for CopyEntityBatchQuery<'a> {
             }
             (false, false) => out.push_sql(BLOCK_RANGE_COLUMN),
         }
+
+        match (self.src.has_causality_region, self.dst.has_causality_region) {
```

Comment:
Not sure if I missed it but I don't think these are now covered in the database tests other than the integration test that uses the ipfs SG right? Can you please ensure this is covered on the store tests? Apologies if it is and I missed it

Reply:
Yes, all these graft combinations should be tested. But at this point this has been open for too long and I'm thinking of merging it without these test, with the risk that there may be a bug in grafting a subgraph with file data sources.